src/resource/cluster.rs

```diff
@@ -15,7 +17,8 @@ resource! {
         /// The current status of this cluster.
         pub status: GenericStatus,
 
-        pub clusters: # FIXME this is just a json blob, we'll examine later
+        /// Details about the clusters that BigML found.
+        pub clusters: Option<serde_json::Value>,
```

Comment:
Seriously, `serde_json::Value` is usually a total nuisance to use in Rust code. It's actually easier to just declare a struct containing only the fields you want to access, and to ignore the rest. I'll take care of it.